components/RoutesPage/index.js

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from "react";
+import RouteCard from "../RouteCard";
+import RouteForm from "../RouteForm";
+import { List, ListItem } from "../RouteList/RouteList.styled";
+
+export default function RoutesPage() {
+  const [myRoutes, setMyRoutes] = useState([]);
+
+  useEffect(() => {
+    fetchRoutes(); // Daten beim Initialisieren der Komponente abrufen
+  }, []);
+
+  async function fetchRoutes() {
+    try {
+      const response = await fetch("/api/routes");
+      const data = await response.json();
+      setMyRoutes(data.reverse()); // Umkehren der Route List
+    } catch (error) {
+      console.error("Failed to fetch routes:", error);
+    }
```

Comment:
I saw you using the useSWR hook in your code already. Try to implement it consistently